cluster/gce/gci/configure-helper.sh

```diff
@@ -2284,6 +2284,20 @@ function setup-addon-manifests {
   fi
 }
 
+# A helper function for setting up addon manifests from outside of k/k
+#
+# $1: addon category under /etc/kubernetes
+# $2: manifest source dir
+# $3: remote manifest repository
+function setup-addon-manifests-remote {
+  local -r dst_dir="/etc/kubernetes/$1/$2"
+  local -r local_addon_dir="/tmp/$2"
+
+  git clone $3 ${local_addon_dir}
```

Comment:
```suggestion
  git clone "$3" "${local_addon_dir}"
```

https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/101039/pull-kubernetes-verify/1403316287957897216#1:build-log.txt%3A2160